policy/daemon_fetchPolicy_test.go

```diff
@@ -0,0 +1,598 @@
+/*
```

Comment:
why separate file?

Reply:
1. test file too long, easy to crash IDE
2. too many test cases in single method and each test case is very long..., difficult to read and write

Reply:
but it is harder to maintain,

Reply:
combine later